seedfile_shift.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
add a short comment explaining what is this script doing.
I would also move it into the helpers folder